code/datums/riding.dm

```diff
@@ -34,16 +32,29 @@
 /datum/riding/proc/force_dismount(mob/living/M)
 	ridden.unbuckle_mob(M)
 
-//Override this to set your vehicle's various pixel offsets
-//if they differ between directions, otherwise use the
-//generic variables
 /datum/riding/proc/handle_vehicle_offsets()
+	var/ridden_dir = "[ridden.dir]"
```

Comment:
the definitions are written as "[NORTH]", so to compare NORTH to"[NORTH]" you have to stringify it.